src/tasks/installers/olm.ts

```diff
@@ -174,22 +179,22 @@ export class OLMTasks {
         title: 'Approve installation',
         enabled: ctx => ctx.approvalStarategy === 'Manual',
         task: async (ctx: any, task: any) => {
-          await kube.approveOperatorInstallationPlan(ctx.installPlanName, flags.chenamespace)
+          await kube.approveOperatorInstallationPlan(ctx.installPlanName, ctx.operatorNamespace)
           task.title = `${task.title}...done.`
         },
       },
       {
         title: 'Wait while operator installed',
         task: async (ctx: any, task: any) => {
-          await kube.waitUntilOperatorIsInstalled(ctx.installPlanName, flags.chenamespace)
+          await kube.waitUntilOperatorIsInstalled(ctx.installPlanName, ctx.operatorNamespace)
           task.title = `${task.title}...done.`
         },
       },
       {
         title: 'Set custom operator image',
         enabled: () => flags['che-operator-image'],
-        task: async (_ctx: any, task: any) => {
-          const csvList = await kube.getClusterServiceVersions(flags.chenamespace)
+        task: async (ctx: any, task: any) => {
```

Comment:
Sorry, here is you renamed _ctx to ctx, but didn't used this variable. Is it ok?